src/scss/app.scss

```diff
@@ -40,31 +22,6 @@ $hover-black: #000;
     transform: scale(0.5);
 }
```

Comment:
In OOCSS, we shouldn't really be styling things by `id`. Is there a way to style this with a set of appropriate classes?